src/main/java/io/appium/java_client/android/Activity.java

```diff
@@ -0,0 +1,193 @@
+package io.appium.java_client.android;
+
+/**
+ * This is a simple POJO class to support the {@link StartsActivity}.
+ */
+public class Activity {
+    private String appPackage;
+    private String appActivity;
+    private String appWaitPackage;
+    private String appWaitActivity;
+    private String intentAction;
+    private String intentCategory;
+    private String intentFlags;
+    private String optionalIntentArguments;
+    private boolean stopApp;
+
+    public Activity(String appPackage, String appActivity) {
+        this.appPackage = appPackage;
+        this.appActivity = appActivity;
+        this.stopApp = true;
+    }
+
+    /**
+     * Gets the app package value.
+     *
+     * @return The app package value.
+     */
+    public String getAppPackage() {
+        return appPackage;
+    }
+
+    /**
+     * Sets the app package value.
+     *
+     * @param appPackage The app package value.
+     */
+    public void setAppPackage(String appPackage) {
```

Comment:
I like the idea behind this but have few things to be discussed here,

Is there a way to remove getters & setters? May be we need to come up with either custom annotation or something like https://github.com/rzwitserloot/lombok.